components/climate/thermostat.rst

```diff
@@ -315,6 +315,93 @@ Set Point Options/Behavior
 **If configured, at least one of** ``default_target_temperature_low`` **and** ``default_target_temperature_high``
 **must be specified in the away mode configuration.**
 
+Presets
+*******
+
+Presets allow you to preconfigure different modes for your thermostat. These will show up in the Home Assistant
+UI where they can change temperature set points, fan modes, fan swing modes, and operating modes all with one
+operation.
+
+- **preset**: (*Optional*, list)
+
+  - **name** (*Required*, string): Name of the preset. If this is one of the *standard* presets (``eco``, ``away``, 
+    ``boost``, ``comfort``, ``home``, ``sleep``, or ``activity``) it is considered a *standard_*preset. Any other
+    string will make the preset a *custom* preset. *Standard* and *custom* presets are functionally equivalent
```

Comment:
More markdown I think. 😄 Also, there should be a comma after the word "equivalent".